pkg/store/container/status.go

```diff
@@ -94,10 +94,15 @@ type Status struct {
 	// Removing indicates that the container is in removing state.
 	// This field doesn't need to be checkpointed.
 	Removing bool `json:"-"`
+	// Unknown indicates that the container status is not fully loaded.
+	Unknown bool `json:"-"`
 }
 
 // State returns current state of the container based on the container status.
 func (s Status) State() runtime.ContainerState {
+	if s.Unknown {
+		return runtime.ContainerState_CONTAINER_UNKNOWN
+	}
```

Comment:
If I'm reading the code right line 115 is an error now, or should not happen right?

Reply:
Adding an error seems to be an overkill, and panic seems to be a bit dangerous.

Reply:
agreed.. so error that shouldn't happen but "handled" error nonetheless since it should only be a checkpoint problem, not an actual container issue? More thinking out loud than suggesting it be a panic or pushing the ignore to the callers.